src/labels/CustomLabels.labels

```diff
@@ -5642,7 +5650,7 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <categories>Gift Entry</categories>
         <language>en_US</language>
         <protected>true</protected>
-        <shortDescription>Gift Entry Update</shortDescription>
+        <shortDescription>Update</shortDescription>
```

Comment:
@kenlewis-sfdo With this updated to a more general description, should (or can) we also update the fullName to be commonUpdate?

Reply:
Thanks, @melissabarber !  Just updated this.